### PR TITLE
Add footer

### LINKED ISSUE #127: footer
was thinking author and license info, maybe another link to the github, similar to our landing page (reuse the design)

### PR DESCRIPTION
**Pull request by SaraVieira**
closes https://github.com/axodotdev/oranda/issues/127

example
CleanShot 2023-01-27 at 20 01 25@2x

fringe pr https://github.com/axodotdev/turbo-axo/pull/195

### CODE REVIEW

**Review by ashleygwilliams — CHANGES_REQUESTED** [review]
i think we'd need to make the copyright configurable as well- e.g. this says copyright oranda, but it's copyright Axo Developer Co. other projects have even more complicated copyright... e.g. https://github.com/rust-lang/rust/blob/master/COPYRIGHT

**Comment by SaraVieira** [comment]
@ashleygwilliams added tests and removed the copyright

File: tests/mod.rs [hunk 1]
```diff
@@ -175,3 +175,13 @@ fn creates_nav() {
...
 }
+
+#[test]
+fn creates_footer() {
+    let _guard = TEST_RUNTIME.enter();
+    let site = Site::build(&config(), &config().readme_path).unwrap();
+
+    println!("{:?}", site.html);
+
+    assert!(site.html.contains("<footer class=\"axo-gradient flex items-center justify-between px-4 py-2 text-slate-50 text-xs w-full\"><span>2023, My Axo project</span></footer>"));
```
Comment (ashleygwilliams) [inline tests/mod.rs]: in a year this test will fail haha

**Review by ashleygwilliams — APPROVED** [review]
approving, but it's possible we may want to remove the year- at least from the test

File: tests/mod.rs (hunk 1 above)
Comment (SaraVieira) [reply tests/mod.rs]: removed the year

**Pull request merged**